src/autopas/pairwiseFunctors/CellFunctor.h

```diff
@@ -175,16 +184,35 @@ template <class Particle, class ParticleCell, class ParticleFunctor, DataLayoutO
           bool bidirectional>
 void CellFunctor<Particle, ParticleCell, ParticleFunctor, DataLayout, useNewton3, bidirectional>::processCellAoSN3(
     ParticleCell &cell) {
-  auto outer = getStaticCellIter(cell);
-  for (; outer.isValid(); ++outer) {
-    Particle &p1 = *outer;
+  if (cell.numParticles() > 8) {
```

Comment:
Replace magic number by a `constexpr` that is documented.
Also, document that this if decides whether or not the sorting is applied since it spans many lines.